Add "private" and "wip" states to Change (Continued)

darfink opened the pull request:
Since the previous pull request (#73) became stale, I've rebased the changes onto latest master, with a spelling correction and a minor correction for the additional tests.

rsandell commented on src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/attr/Change.java:
```diff
@@ -144,6 +147,16 @@ private Object readResolve() {
+    /**
+     * Is this change private.
+     */
+    private boolean isPrivate;
```
doesn't checkstyle or findbugs have an issue with this name?

rsandell replied:
Yeah, I don't like the name either :) Understandable that you can't use `private` since that's reserved. But I guess `_private` in that case would be better?

darfink commented:
I never did run `checkstyle` since it wasn't mentioned in the README. Anyhow, a few whitespace issues were detected but I've fixed them. Checkstyle did complain about `isPrivate` argument hiding the corresponding field. I've added a ignore directive but you may prefer another solution.

cashlalala commented:
Vote for this PR! Look forward to have it in the near future! :)

darfink commented:
@rsandell any more input on this?

cashlalala commented on src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/events/WipStateChanged.java:
```diff
@@ -0,0 +1,43 @@
+public class WipStateChanged extends ChangeBasedEvent {
+  @Override
+  public GerritEventType getEventType() {
+    return GerritEventType.PRIVATE_STATE_CHANGED;
```
@darfink  shouldn't this line be GerritEventType.WIP_STATE_CHANGED?

darfink replied:
It most certainly should, great catch!

cashlalala commented:
:) thx
